core/tests/protractor_utils/AdminPage.js

```diff
@@ -260,23 +262,23 @@ var AdminPage = function() {
   this.getUsersAsssignedToRole = async function(role) {
     await waitFor.visibilityOf(
       roleDropdown, 'View role dropdown taking too long to be visible');
-    await roleDropdown.sendKeys('By Role');
+    await action.sendKeys('Role Drop Down', roleDropdown, 'By Role');
 
-    await roleValueOption.click();
-    await roleValueOption.sendKeys(role);
+    await action.click(roleValueOption);
+    await action.sendKeys('Role Value Option', roleValueOption, role);
 
-    await viewRoleButton.click();
+    await action.click(viewRoleButton);
   };
 
   this.viewRolesbyUsername = async function(username) {
     await waitFor.visibilityOf(
       roleDropdown, 'View role dropdown taking too long to be visible');
-    await roleDropdown.sendKeys('By Username');
+    await action.sendKeys('Role Drop Down', roleDropdown, 'By Username');
 
-    await roleUsernameOption.click();
-    await roleUsernameOption.sendKeys(username);
+    await action.click(roleUsernameOption);
+    await action.sendKeys('Role Username Option', roleUsernameOption, username);
 
-    await viewRoleButton.click();
+    await action.click(viewRoleButton);
   };
 
   this.expectUsernamesToMatch = async function(expectedUsernamesArray) {
```

Comment:
Below, there are more `getText()` calls that need `waitFor` statements:
* https://github.com/oppia/oppia/pull/10831/files#diff-1031f0f30ce6a90c24332497a575023cR289
* https://github.com/oppia/oppia/pull/10831/files#diff-1031f0f30ce6a90c24332497a575023cR389
* https://github.com/oppia/oppia/pull/10831/files#diff-1031f0f30ce6a90c24332497a575023cR400